### ISSUE TITLE
Shiny development

### ISSUE THREAD

**Issue by Prof-ThiagoOliveira** [DidierMurilloF/FielDHub#11]
In the summary of your manuscript, you describe: "One of the problems that life scientists often face is the lack of freely available and user-friendly interactive tools to create designs that fit their needs". I absolutely agree with the author's phrase. Thus, I proposed here a discussion about this topic. 

By user-friendly definition, I understand that common attributes of it should be
 1. **Interface should be simple/straightforward**. It should provide quick access to common features or commands.
 2. **Clean**. A good user interface is well-organized (users cannot feel confounded among inputs)
 3. **Intuitive**. The interface must make sense to the average user and require a minimal explanation for using it.
 4. **Reliable**.  It is something that doesn't cause undue frustration for the user. 

Let us discuss each of these items now. 
  1. The shiny interface that the authors provided looks nice, and the inclusion of a box with all inputs is well defined. It is quite easy for the end-user to interact with the dashboard, and it puts the app as a simple/straightforward tool to create designs.
  2. Although the authors proposed some complex experimental designs, their app seems very clean and with an organized interface. 
  3. Unfortunately, some inputs of experimental designs proposed in the app are not intuitive and require some improvement to the end-user to understand the whole idea. I can use myself as an example here. I have never worked with the unreplicated design before be presented to this manuscript/package. In this way, I can tell the authors that a student or people with no background won't use this interface because it is too hard to set up each input means. This is due to the app didn't provide any description of each input. Consequently, the user needs to kind of guess the meaning of each one. To turn the app intuitive, I suggest that the authors work a little more in the description step and perhaps include a concise introduction to each design (say one paragraph).
  4. As the app is sometimes not intuitive, it also causes frustration for the user. However, it is reliable because it does not malfunction or crash.

With these improvements, the application will greatly impact universities and be something easy to use.

**Comment by tatirri** [comment]
Dear Thiago,

Thank you for the careful review and relevant suggestions. We have put a lot of effort into these areas and appreciate the feedback. We are actively working to build more interactive guides for more complex experimental designs. We estimate that these changes will take weeks to properly design and incorporate, and we are planning to include them in the next release. As users are actively planning their experiments for 2021, we feel that FielDHub is sufficient for meeting users needs for this upcoming growing season. Delaying release will prevent some of our users from designing their experiments in FielDHub this year. While we remain focused on improving usability and will add more intuitive guides, we believe that it is helpful for users this year and we are seeking to distribute FielDHub as soon as possible. The less complex designs, such as RCBD, are the most widely used, and FielDHub is simple for those use cases. Nevertheless, we are willing to remove references to an intuitive design until we update the guides for the most complex designs if that helps us to expedite publication.

Didier, Ana & Tom

**Comment by Prof-ThiagoOliveira** [comment]
Dear Didier, Ana & Tom, 

I absolutely agree with your point of view. There is no reason to barrier the publication due to small (but important) details for the first release. I agree that this suggestion will take some weeks to be included, and I am more than happy to accept the paper when other suggestions I have made were implemented in the package. We can leave this suggestion for the next release. I kindly ask you to make the next realise as soon as possible. 

Is this reasonable for you?
Best wishes, 

Thiago
